src/Microsoft.OpenApi.Readers/Microsoft.OpenApi.Readers.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <LangVersion>9.0</LangVersion>
```

Comment:
why are we restricting the lang version here?

Reply:
The `static` anonymous function is unavailable in C# 7.3 which is what the project was using.

Reply:
why not set this to latest instead?

Reply:
Resolving..